Tests/FunctionalTests/ModuleAliasingTests.swift

```diff
@@ -29,9 +32,16 @@ class ModuleAliasingTests: XCTestCase {
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "release", "GameUtils.swiftmodule"))
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "debug", "Utils.swiftmodule"))
             XCTAssertFileExists(prefix.appending(components: "AppPkg", ".build", UserToolchain.default.triple.platformBuildPathComponent(), "release", "Utils.swiftmodule"))
+            
             let result = try SwiftPMProduct.SwiftBuild.executeProcess([], packagePath: app)
-            XCTAssertEqual(result.exitStatus, .terminated(code: 0))
+            let output = try result.utf8Output() + result.utf8stderrOutput()
+           
+            if result.exitStatus != 0 {
+                // FIXME: rdar://88722540
+                // The process from above crashes in a certain env, so print
+                // the output for further investigation
+                try XCTSkipIf(true, output)
```

Comment:
Somewhat of a nitpick but for anyone else who looks at these logs, it would be great if the message was something like: `skipping due to expected failure being investigated in rdar://88722540\nresult: \(result.exitStatus)\noutput: \(output)`.  I know that's a bit verbose but then if others are looking at failures it's clear what's going on and a reference to the Radar.  Plus it tells us the exit status, in case it suddenly changes from signal 11 to something else.

Reply:
I think you could even do `XCTSkipIf(result.exitStatus != 0, "...")` but maybe that's overthinking this.  The comment is still useful though.

Reply:
I'm saying this partly because there have been cases where printed output from our tests that had "error:" at the start of a line was found by people grepping the logs and caused some confusion (the tests was that a failure occurred and _expected_ the "error:" message to be there).

Reply:
So clarity is probably good here for anyone looking at logs who has never heard of this particular failure.